src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -127,25 +128,25 @@ public void parseAddress_validValueWithWhitespace_returnsTrimmedAddress() throws
 
     @Test
     public void parseEmail_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> ParserUtil.parseEmail((String) null));
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseEmail((Optional<String>) null));
```

Comment:
Is this type casting needed?

Reply:
Yes ParseEmail takes in Optional<String> type only.